demo/client/backend/config.json

```diff
@@ -0,0 +1,14 @@
+{
+    "connection_profile_filename":"../gateway/localhost/connection_profile.json",
+    "channel_name":"mychannel",
```

Comment:
btw: my PR for docker-compose makes now channel name configurable if you define the CHANNEL_NAME env variables (of similar interest might be NODE_WALLETS for your wallet below; would make sure that you don't have left-over inconsistent wallets if you remove CA state with teardown --clean-slate  ...

Reply:
> btw: my PR for docker-compose makes now channel name configurable if you define the CHANNEL_NAME env variables (of similar interest might be NODE_WALLETS for your wallet below; would make sure that you don't have left-over inconsistent wallets if you remove CA state with teardown --clean-slate ...

currently, the code uses config.json for all configuration.    I prefer to go with either env variables or config.json.   
I propose to postpone this change for a future PR so that we have all the configuration parameters (and their usage) and we can make _one_ change to env variables, if needed.

Reply:
this was mostly an FYI that the docker-compose network has now a configurable CHANNEL_NAME, so once you get there to configure it from config.json, you just would have to set the CHANNEL_NAME env-variable to whatever there is in config.json before calling the scripts from utils/docker-compose and they should use the correct channel name. Definitely did _not_ mean to do all of this in this PR.